client/webserver/site/cache_utilities.bash

```diff
@@ -5,14 +5,20 @@ CSS_FILE=${SITE_DIR}/dist/style.css
 JS_DIR=${SITE_DIR}/src/js
 JS_FILE=${SITE_DIR}/dist/entry.js
 
-hashfile () { sha1sum $1 | cut -d " " -f1 ; }
+if [[ "$OSTYPE" == "darwin"* ]]; then
+   hashfunc="shasum"
+else
+   hashfunc="sha1sum"
+fi
```

Comment:
Some of the decred folks seem to prefer using openssl as a portable approach.  `openssl sha1 -r blah.txt`
The `sha*sum` functions seem be be part of typical "coreutils" packags in NIX systems.  I've always felt like that's more common than having openssl installed personally, but it does make the app consistent so there's no question about syntax or output format.

Reply:
> I wonder is openssl available out of the box on Mac?

yh, but the latest macOS, `openssl version` shows Apple uses another flavour `LibreSSL`.